src/wallet/wallet.cpp

```diff
@@ -4145,3 +4148,14 @@ bool CMerkleTx::AcceptToMemoryPool(const CAmount& nAbsurdFee, CValidationState&
 {
     return ::AcceptToMemoryPool(mempool, state, tx, true, NULL, NULL, false, nAbsurdFee);
 }
+
+bool CalculateEstimateType(FeeEstimateMode mode) {
```

Comment:
Please see #10706 
CalculateEstimateType is now only used one place, in GetMinimumFee.